CHANGELOG.md

```
@@ -1,5 +1,21 @@
+## [Unreleased]
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)